server/api/middlewares/validateS2Stoken.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response, NextFunction } from 'express';
+import config from 'config';
+import { ERROR_CLIENT_NOT_WHITELISTED, ERROR_NO_S2S_TOKEN } from '../errors';
+import { getTokenDetails } from '../services/tokenDetails.service';
+
+// Check if S2S token contains 'Bearer', if not , add it.
+export const formatS2SToken = (token: string): string => {
```

Comment:
This seems wrong. As I understand it an s2s token shouldn't start with bearer

Reply:
This is just a refactor of the S2S branch that @borislavpetrovhmcts created. But I do believe the rpe service does fail if bearer is not passed in.